src/presign/round_one.rs

```diff
@@ -72,7 +72,7 @@ impl Public {
             broadcasted_params,
         ) {
             Ok(()) => Ok(round_one_public),
-            Err(e) => bail!("Failed to verify round one public: {}", e),
+            Err(e) => Err(e),
```

Comment:
This match can be further simplified as:
```rust
s can be further simplified as just:
```
round_one_public.verify(
&receiver_keygen_public.params,
&sender_keygen_public.pk,
broadcasted_params,
)?;
Ok(round_one_public)
```
```